pages/Parser.md

```diff
@@ -4,13 +4,13 @@ title: Open API parser | Hydraecosystem.org
 permalink: /Parser/
 ---
 
-# Open API to Hydra Parser 
+# Open API to [Hydra Parser](https://github.com/HTTP-APIs/hydra-openapi-parser/) 
```

Comment:
I wanted to suggest that instead of too many links pointing to the same resource, a single more contextual link will help with understanding. Something like in #53